.github/workflows/testing.yml

```diff
@@ -1,35 +1,51 @@
 name: Pytest MSS
 
 on: [ push, pull_request ]
+env:
+  PAT: ${{ secrets.PAT }}
 
 jobs:
   Test-MSS:
     runs-on: ubuntu-latest
```

Comment:
what is the meaning of this? Are we not using our own container? Is this the environment within which our docker is executed?

Reply:
We need an environment first, before we can execute a docker container in it.
So your second assumption is correct.